MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
```

Comment:
I would make this a protocol and on `registerCellClass` use only one parameter that is a `UITableViewCell` and conforms to this protocol.
What do you think?